uk.co.compucorp.civicrm.civihr/README.md

```diff
@@ -0,0 +1,20 @@
+CiviHR extension
+======
+
+This extension provides informations about current CiviHR release.
+
+It contains CRM_CiviHR class with static methods listed below:
+* _getVersion()_ - returning current CiviHR version defined in info.xml file
+
+Also it provides CiviHR entity API entry points listed below:
+* _getversion_ - returning current CiviHR version defined in info.xml file
+
+Installation
+------
+
+Go to 
+- Administer -> System Settings -> Manage Extensions
+
+and install Activity Report (uk.co.compucorp.civicrm.civihr) extension.
```

Comment:
you mistakenly wrote `and install Activity Report`  but this extension is called currently "CiviHR"